### PR TITLE
Created a page for the Backend Career Roadmap which contains information about backend web development and links to useful resources on programming with Python, Mysql, as well as creating and utilizing APIs and Databases

### PR DESCRIPTION
**Pull request by peteradeojo**
## Changes:
- [x] I added a new Roadmap
- [ ] I updated an existing roadmap

## Description:

(What Roadmap did you work on)
Backend
(Describe how this update/changes were implemented)
I copied the frontend roadmap to use a layout template and then updated, replaced and added relevant information and links to resources. I also updated the sidemenu to reflect correctly the page content

## Checklist:

- [x] I have confirmed that this is not a duplicate Roadmap
- [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/Timonwa/techroadmap/pulls) for the same update/change
- [x] My code follows the [style guidelines](https://github.com/Timonwa/techroadmap/blob/main/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Timonwa** [comment]
Thank you @peteradeojo for this. I really appreciate it.

File: src/html/careers/backend.html [hunk 1]
```diff
@@ -0,0 +1,1703 @@
...
+              <li class="resource-link">
+                <a target="_blank" href="https://vercel.com">Website</a>
+                <span class="resource-link-badge">Link</span>
+              </li>
+              <li class="resource-link">
+                <a target="_blank" href="https://vercel.com/docs">Documentation</a>
+                <span class="resource-link-badge">Link</span>
+              </li>
+              <li class="resource-link">
+                <a target="_blank" href="https://youtu.be/mP8KIAros70">Deploy a website to vercel. - WebDemystified</a>
```
Comment (humaneguy) [inline src/html/careers/backend.html:910]: This seems to be a frontend kind of link.
Vercel support backend languages using [Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions)

File: src/html/careers/backend.html [hunk 2]
```diff
@@ -0,0 +1,1703 @@
...
+                <span class="resource-link-badge">Video</span>
+              </li>
+              <li class="resource-link">
+                <a target="_blank" href="https://youtu.be/6YRymPx1-4s">Deploying an application with vercel. - Caleb
```
Comment (humaneguy) [inline src/html/careers/backend.html:914]: Same as this.

Comment (humaneguy) [reply src/html/careers/backend.html:914]: For more backend deployments. Checkout

1. [Digtal Ocean](https://www.digitalocean.com/)
2. [Amazon Web Services](https://aws.amazon.com/)
3. [Google Cloud Platform](https://cloud.google.com/)

Comment (humaneguy) [reply src/html/careers/backend.html:914]: These platforms provide tutorials for deployment.

File: src/html/careers/backend.html [hunk 3]
```diff
@@ -0,0 +1,1703 @@
...
+
+          <!-- others -->
+          <div class="resource-card">
+            <h3 class="resource-card-title">Others</h3>
+            <p class="resource-card-body">
+              Tips and tricks, tools, cool examples etc.
+            </p>
+            <ul class="resource-card-links">
+              <li class="resource-link">
+                <a target="_blank" href=""></a>
```
Comment (humaneguy) [inline src/html/careers/backend.html:1269]: An empty item?

Comment (humaneguy) [reply src/html/careers/backend.html:1269]: You can remove this.

**Review by humaneguy — COMMENTED** [review]
Really great work, tho'.
I've appended some little comments you might want to check out.

File: src/html/careers/backend.html [hunk 4]
```diff
@@ -0,0 +1,1703 @@
...
+          <div id="internships" class="resource-card">
+            <h3 class="resource-card-title">Internships</h3>
+            <ul class="resource-card-links">
+              <li class="resource-link">
+                <a target="_blank" href="https://twitter.com/hnginternship?t=FNo8Fg7sQj53daEh0mCuew&s=09">HNG
+                  Internship</a>
+                <span class="resource-link-badge">Unpaid</span>
+              </li>
+              <li class="resource-link">
+                <a target="_blank" href="https://internshala.com/">Internshala</a>
```
[the remark this answers is not in the document]
Comment (Timonwa) [reply src/html/careers/backend.html:488]: Paid means you get paid stipends/salary.

File: src/html/careers/backend.html [hunk 5]
```diff
@@ -0,0 +1,1703 @@
...
+              <li class="resource-link">
+                <a target="_blank" href="https://www.freecodecamp.org/news/learn-backend-development/">The Best Way to
+                  Learn Backend Web Development - FreeCodeCamp.org</a>
+              </li>
+              <li class="resource-link">
+                <a target="_blank" href="https://www.interviewbit.com/blog/backend-developer-skills/">Backend Developer
+                  Skills You Must Have - InterviewBit.com</a>
+              </li>
+              <li class="resource-link">
+                <a target="_blank" href="https://learntocodewith.me/posts/backend-development/">The Beginner's Guide to
```
Comment (Timonwa) [inline src/html/careers/backend.html:368]: this link should come first

File: src/html/careers/backend.html [hunk 6]
```diff
@@ -0,0 +1,1703 @@
...
+        <div class="resource-section__links">
+          <!-- articles -->
+          <div class="resource-card">
+            <h3 class="resource-card-title">Articles</h3>
+            <ul class="resource-card-links">
+              <li class="resource-link">
+                <a target="_blank" href="https://www.freecodecamp.org/news/learn-backend-development/">The Best Way to
+                  Learn Backend Web Development - FreeCodeCamp.org</a>
+              </li>
+              <li class="resource-link">
```
Comment (Timonwa) [inline src/html/careers/backend.html:363]: this link should come second.

File: src/html/careers/backend.html [hunk 7]
```diff
@@ -0,0 +1,1703 @@
...
+          </p>
+        </div>
+
+        <div class="resource-section__links">
+          <!-- articles -->
+          <div class="resource-card">
+            <h3 class="resource-card-title">Articles</h3>
+            <ul class="resource-card-links">
+              <li class="resource-link">
+                <a target="_blank" href="https://www.freecodecamp.org/news/learn-backend-development/">The Best Way to
```
Comment (Timonwa) [inline src/html/careers/backend.html:360]: this link should come third

File: src/html/careers/backend.html (hunk 5 above)
Comment (Timonwa) [reply src/html/careers/backend.html:368]: also the name of the  author/webiste is missing in the description.

File: src/html/careers/backend.html [hunk 8]
```diff
@@ -0,0 +1,1703 @@
...
+                <span class="resource-link-badge">Article</span>
+              </li>
+              <!-- <li class="resource-link">
+                  <a target="_blank" href="https://www.freecodecamp.org/news/javascript-projects-for-beginners">40
+                    JavaScript projects for beginners – Easy ideas to get
+                    started coding JS. ~ FreeCodeCamp</a>
+                  </li> -->
+              <li class="resource-link">
+                <a target="_blank" href="https://www.youtube.com/watch?v=-MTSQjw5DrM">40
+                  RESTFU APIs in 100 seconds // Build an API from Scratch with Node.js Express ~ Fireship</a>
```
Comment (Timonwa) [inline src/html/careers/backend.html:1199]: Remove this link. It should be In the API section instead, as the first link in the video section

File: src/html/careers/backend.html [hunk 9]
```diff
@@ -0,0 +1,1703 @@
...
+              development.</p>
+
+            <ul class="resource-card-links">
+              <li class="resource-link">
+                <a href="https://www.mongodb.com" target="_blank" rel="noopener noreferrer">Website</a>
+                <!-- <span class="resource-link-badge">Video</span> -->
+              </li>
+              <li class="resource-link">
+                <a href="https://www.youtube.com/watch?v=bxsemcrY4gQ" target="_blank" rel="noopener noreferrer">Node.js
+                  Crash Course Tutorial #9</a>
```
Comment (Timonwa) [inline src/html/careers/backend.html:1323]: Remove this link as it's not focused on  MySql

File: src/html/careers/backend.html [hunk 10]
```diff
@@ -0,0 +1,1703 @@
...
+          </div>
+
+          <!-- others -->
+          <div class="resource-card">
+            <h3 class="resource-card-title">Others</h3>
+            <p class="resource-card-body">
+              Tips and tricks, tools, cool examples etc.
+            </p>
+            <ul class="resource-card-links">
+              <li class="resource-link">
```
Comment (Timonwa) [inline src/html/careers/backend.html:1422]: Remove all the links here except the one about Postman. The rest are for frontend devs and not backend de

File: src/html/careers/backend.html [hunk 11]
```diff
@@ -0,0 +1,1703 @@
...
+              </li>
+              <li class="resource-link">
+                <a target="_blank" href="https://www.youtube.com/watch?v=L72fhGm1tfE">Express JS crash course. ~
+                  Traversy Media</a>
+                <span class="resource-link-badge">Video</span>
+              </li>
+              <li class="resource-link">
+                <a target="_blank"
+                  href="https://www.youtube.com/watch?v=KtnHb7FMk2Q&list=PLp50dWW_m40Vruw9uKGNqySCNFLXK5YiO">Express JS
+                  Tutorial for Beginners. ~ ARC Tutorials</a>
```
Comment (Timonwa) [inline src/html/careers/backend.html:1488]: Add the direct link to the playlist. And let the badge say **Playlist**

File: src/html/careers/backend.html [hunk 12]
```diff
@@ -0,0 +1,1703 @@
...
+                <span class="resource-link-badge"></span>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <!--* Javascript -->
+      <section id="javascript" class="resource-section">
+        <h2 class="resource-section__heading">Javascript</h2>
```
Comment (Timonwa) [inline src/html/careers/backend.html:1083]: Update the name to JavaScript(Node.js)

**Comment by Timonwa** [comment]
@all-contributors please add @peteradeojo for roadmaps ,code and resources

**Comment by Timonwa** [comment]
@all-contributors please add @peteradeojo for  code and data

**Pull request merged**